packages/from-to-location-picker/src/index.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { IntlProvider } from "react-intl";
+import { action } from "@storybook/addon-actions";
+
+import FromToLocationPicker from ".";
+
+import "../__mocks__/trimet-mock.css";
+
+export default {
+  title: "From-To-Picker",
+  component: FromToLocationPicker
+};
+
+const onFromClick = action("onFromClick");
+const onToClick = action("onToClick");
+
+export const fromTo = (): React.ReactElement => (
+  <IntlProvider locale="en-US">
+    <FromToLocationPicker onFromClick={onFromClick} onToClick={onToClick} />
+  </IntlProvider>
+);
+
+export const smallTextSansSerif = (): React.ReactElement => (
+  <IntlProvider locale="en-US">
+    <span style={{ fontSize: "75%", fontFamily: "sans-serif" }}>
+      Plan a trip:
+      <FromToLocationPicker onFromClick={onFromClick} onToClick={onToClick} />
+    </span>
+  </IntlProvider>
+);
+
+export const otpAmbientStyle = (): React.ReactElement => (
```

Comment:
Rename this story to say "custom style and text".

Reply:
Good call! Was wondering what ambient style meant. Renamed in 57e3522f13c3e0c11b4eef871ff4200451264d54